content/en/synthetics/troubleshooting/_index.md

```diff
@@ -141,6 +141,12 @@ Additionally, you might also have to ensure [Datadog Synthetic Monitoring IP ran
 
 Synthetic tests by default do not [renotify][12]. This means that if you add your notification handle such as your email address or Slack handle after a transition is generated (for example: a test going into alert or recovering from a previous alert), a notification is not sent for that transition. A notification is sent for the next transition.
 
+## Mobile tests
+
+### Unable to launch a device recording
+
+If there are security checks during app startup iIn particular if USB debugging is checked), it would be required to upload an .IPA that does not contain these checks. 
```

Comment:
```suggestion
If there are security checks during app startup (in particular if USB debugging is checked), it would be required to upload an .ipa that does not contain these checks. 
```
Fixing a typo and case on .ipa: I uppercased in on Slack but I realize in all of the docs (and in the filesystem) it's lowercase https://github.com/DataDog/documentation/blob/8070cb0b6ce07ca0c8bfe7319ab06eee51e9b677/content/en/synthetics/mobile_app_testing/settings/_index.md?plain=1#L76